core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.html

```diff
@@ -1,5 +1,4 @@
-<div class="texera-workspace-workflow-editor-body texera-workflow-component-body">
-  <p>
-    workflow-editor works!
-  </p>  
+<div class="texera-workspace-workflow-editor-body texera-workflow-component-body" [attr.id]="WORKFLOW_EDITOR_JOINTJS_WRAPPER_ID">
+  <div class ="texera-workspace-workflow-jointjs" [attr.id]="WORKFLOW_EDITOR_JOINTJS_ID">
```

Comment:
attr.id is basically the same thing with doing <div #id="WORKFLOW_EDITOR_JOINTJS_WRAPPER_ID"></div>

we uses attr.id because we want to use the variable defined in the workflow-editor.component.ts. Since resizing will also need the ID of the paper, we decided that it is better to defined the ID in the TS file to improve the structure of the code base.